plugins/modules/swim_intent.py

```diff
@@ -499,32 +504,31 @@ def get_image_id(self, name):
             function='get_software_image_details',
             params={"image_name": name},
         )
-
-        self.log(str(image_response))
-
+        self.log("Received API response from 'get_software_image_details': {0}".format(str(image_response)), "DEBUG")
         image_list = image_response.get("response")
+
         if (len(image_list) == 1):
             image_id = image_list[0].get("imageUuid")
-            self.log("Image Id: " + str(image_id))
+            self.log("SWIM image {0} having the Id: {1}".format(name, image_id), "INFO")
```

Comment:
self.log("SWIM image '{0}' has the ID: {1}".format(name, image_id), "INFO")

Reply:
added.